posthog/api/feature_flag.py

```diff
@@ -308,6 +310,10 @@ def get_queryset(self) -> QuerySet:
 
         if self.action == "list":
             queryset = queryset.filter(deleted=False).prefetch_related("experiment_set").prefetch_related("features")
+            survey_targeting_flags = Survey.objects.filter(team=self.team, targeting_flag__isnull=False).values_list(
+                "targeting_flag_id", flat=True
+            )
+            queryset = queryset.exclude(Q(id__in=survey_targeting_flags))
```

Comment:
Do we want to leave these flags out vs keeping them specially annotated like experiment flags? Just food for thought I don't know what's best here